.github/workflows/ci.yml

```diff
@@ -49,17 +49,18 @@ jobs:
       fail-fast: true
       matrix:
         os:
-          - { label: ubuntu-20.04, code: focal }
+          - { label: ubuntu-latest, code: latest }
         compiler:
           - { compiler: GNU10,  CC: gcc-10,   CXX: g++-10,     packages: gcc-10 g++-10 }
```

Comment:
Also, newest GCC is v11.

Reply:
Yes I know. In a comment to PR, I wrote an explanation why I left basic GCC, not newest as one of the compilers in Linux-minimal. I don't have a strong opinion on this, though.

In my opinion, it doesn't really matter which version of GCC Linux-minimal runs, whether it's the latest version or the base version. Linux-minimal is created to catch code compilation problems as quickly as possible before running the main matrix, and such problems either occur in all compilers (if it's clearly bad code) or usually only in the latest LLVMs.

IF we decide to consider only the latest Ubuntu LTS (currently 22.04) as officially supported, then GCC 11 can be installed as the oldest GCC to verify the build.